usbsdmux/__main__.py

```diff
@@ -121,11 +122,11 @@ def main():
         elif mode == "info":
             info = ctl.get_card_info()
             if args.json:
-                print(json.dumps(info, indent=4))
+                print(json.dumps(info, indent=2))
```

Comment:
I am fine with a more compact output. I'll add this change to the other JSON outputs in another PR :-)